packages/editor/src/components/post-url/label.js

```diff
@@ -10,6 +10,10 @@ import { filterURLForDisplay } from '@wordpress/url';
 import { store as editorStore } from '../../store';
 
 export default function PostURLLabel() {
+	return usePostURLLabel();
+}
```

Comment:
Should we remove this component? It's not used by core anymore.

Reply:
We can't without breaking backwards compatibility—it's exported from `@wordpress/editor`. I think it's fine to support both a component version and a hook version of these labels. It's a bit like how we have both `withSelect` and `useSelect`.